mantidimaging/gui/windows/main/view.py

```diff
@@ -135,19 +134,19 @@ def __init__(self, open_dialogs: bool = True):
         self.setup_shortcuts()
         self.update_shortcuts()
         self.setAcceptDrops(True)
-        base_path = finder.ROOT_PATH
 
+        base_path = Path(finder.ROOT_PATH)
```

Comment:
I think we can condense this by just setting `image_path` once since the path in the `else` statement ("gui/ui/images/mantid_imaging_64px.png") is the same in both cases:
```python
base_path = Path(finder.ROOT_PATH)
self.open_dialogs = open_dialogs

if self.open_dialogs and versions.is_prerelease():
    self.setWindowTitle("Mantid Imaging Unstable")
    image_path = "gui/ui/images/mantid_imaging_unstable_64px.png"
else:
    image_path = "gui/ui/images/mantid_imaging_64px.png"

bg_image = base_path / image_path
self.setWindowIcon(QIcon(str(bg_image)))
```